index.html

```diff
@@ -0,0 +1,212 @@
+
+    <section class="section section--features">
+
+      <div class="header header--features">
+        <h2 class="section__h1 section__h1--features">FEATURES</h2>
+      </div>
+
+      <div class="section__container section__container--features">
+
+        <div class="section__item">
+          <div class="section__img section__img--features">
+            <img src="./images/responsive.svg" alt="img">
+          </div>
+          <h3 class="section__h2 section__h2--features">FULLY RESPONSIVE</h3>
+          <p class="section__p section__p--features">Looks amazing on any device: smartphone, tablet, laptop and desktop.</p>
+        </div>
+
+        <div class="section__item">
+          <div class="section__img section__img--features">
+            <img src="./images/customizable.svg" alt="img">
+          </div>
+          <h3 class="section__h2 section__h2--features">CUSTOMIZABLE</h3>
+          <p class="section__p section__p--features">Change the colors, pictures or any of the sections to suit your needs.</p>
+        </div>
+        
+        <div class="section__item">
+          <div class="section__img section__img--features">
+            <img src="./images/design.svg" alt="img">
+          </div>
+          <h3 class="section__h2 section__h2--features">SLICK AND BEAUTIFUL DESIGN</h3>
+          <p class="section__p section__p--features">Trendy and fresh design, fits any website.</p>
+        </div>
+
+      </div>
+
+    </section>
+
+  </div>
+
+  <div class="wrapper wrapper--logos">
+    <div class="container container--logos">
+      <img src="./images/logos.png" alt="logos" class="container__img">
+    </div>
+  </div>
+  
+  <div class="wrapper wrapper--iMac">
+
+    <div class="container container--iMac">
+      <img src="./images/imac.png" alt="iMac" class="container__img">
+    </div>
+
+    <section class="section section--iMac">
+      <h2 class="section__h1 section__h1--iMac">SIMPLE AND BEAUTIFUL</h2>
+      <p class="section__p section__p--iMac">Use Nova theme for your next web project. It is completely customizable so you can change any of the sections to fit your needs. Nova Theme is Free for any kind of use, personal and commercial. Have fun and good luck!</p>
+    </section>
+
+  </div>
+
+  <div class="wrapper wrapper--plan">
+
+    <section class="section section--plan">
+
+      <div class="header header--plan">
+        <h2 class="section__h1 section__h1--plan">CHOOSE YOUR PRICING PLAN</h2>
+        <p class="section__p section__p--plan">Pick any of our super affordable pricing plans</p>
+      </div>
+
+      <div class="section__container section__container--plan">
+
+        <div class="section__item section__item--plan">
+          <h3 class="section__h2 section__h2--plan-title">BASIC</h3>
+          <p class="section__p section--price">$15</p>
+          <p class="section__p section--price-info">per month</p>
+          <p class="section__p section--plan-info">Up to 7 Projects<br>2 Additional Developers</p>
```

Comment:
Zdecydowanie każdy z elementów powinien być `ul > li`, bo to lista cech dla danego pakietu.